src/definitions.php

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>EKGViewer - Definitions</title>
+    <link href="./css/reset.css" rel="stylesheet" />
+    <link href="./templates/css/page-template.css" rel="stylesheet" />
+    <link href="./css/definitions.css" rel="stylesheet" />
+</head>
+
+<body>
+    <?php include "./templates/page-header.php" ?>
+    <section id="content-wrapper"></section>
+    <?php include "./templates/page-footer.php" ?>
+</body>
```

Comment:
The body pulls the header and footer from the templates. The main page content goes in the "content-wrapper" section tag.